apps/publikator/components/editor/modules/embeddatawrapper/index.js

```diff
@@ -62,7 +62,10 @@ const EmbedDatawrapper = ({ rule, TYPE }) => {
   const newItem = () =>
     Block.create({
       type: TYPE,
-      isVoid: true
+      isVoid: true,
+      data: {
+        plain: !resizable,
```

Comment:
What does this do? Isn't `plain` its own option, unrelated to resizable?

Reply:
yeah, i just thought maybe the editorial newsletter, since the header/footer of the chart don't resize, they look a bit awkward, so not displaying them by default makes sense…? 🤷‍♂️ 
Unless you think it's a terrible idea in the first place, I'll add a comment in the code to clarify.

Reply:
refactored to make it clearer.